internal/orchestrator/orchestrator_test.go

```diff
@@ -22,6 +22,8 @@ import (
 	"testing"
 
 	"github.com/hyperledger-labs/firefly/internal/config"
+	"github.com/hyperledger-labs/firefly/internal/tokens/tkfactory"
```

Comment:
Should this be `tifactory` for consistency - e.g. it's the `[Ti] Tokens Interface` plug point.

Reply:
Fine by me as long as [Ti](http://www.ti.com) doesn't come for us 😄